src-cpp/ProducerImpl.cpp

```diff
@@ -33,6 +33,16 @@
 
 #include "rdkafkacpp_int.h"
 
+const int RdKafka::Producer::RK_MSG_FREE;
+
+const int RdKafka::Producer::RK_MSG_COPY;
+
+const int RdKafka::Producer::RK_MSG_BLOCK;
+
+#ifndef MSG_COPY
```

Comment:
Right.
Seeing how those two are legacy ones (to be backwards compatible with old appS), do you think we could get away with not declaring them here at all? It is not like those apps will suddenly break because the new constants are now fixed.